helpers/opentracing.go

```diff
@@ -0,0 +1,34 @@
+package helpers
```

Comment:
Could you make call this packing opentracinghelpers? Or opentracingutil? Or tracingutil?